File: FaceSample/app/src/main/java/com/regula/facesample/items/other/VideoCompletionItem.java
package com.regula.facesample.items.other;

import android.content.Context;
import android.util.Log;

import androidx.annotation.NonNull;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;
import com.regula.facesdk.listener.IVideoEncoderCompletion;

public class VideoCompletionItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        FaceSDK.Instance().setVideoEncoderCompletion(new IVideoEncoderCompletion() {
            @Override
            public void onVideoRecorded(@NonNull String transactionId, boolean success) {
                Log.d("VideoCompletionItem", "Transaction video: " + transactionId + " success: " + success);
            }
        });
        FaceSDK.Instance().startLiveness(context, livenessResponse -> { });
    }

    @Override
    public String getDescription() {
        return "Video completion about uploading liveness video to service";
    }

    @Override
    public String getTitle() {
        return "Video completion";
    }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/other/VideoCompletionItem.kt
package com.regula.facesamplekotlin.items.other

import android.content.Context
import android.util.Log
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK

class VideoCompletionItem: CategoryItem() {

    override fun onItemSelected(context: Context) {
        FaceSDK.Instance().setVideoEncoderCompletion { transactionId, success ->
            Log.d("VideoCompletionItem", "Transaction video: $transactionId success: $success")
        }
        FaceSDK.Instance().startLiveness(context) { }
    }

    override val title: String
        get() = "Video completion about uploading\nliveness video to service"

    override val description: String
        get() = "Video completion"
}